src/SharedMoudule/Components/ViewModal/ViewModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { Modal } from 'react-bootstrap'
+
+export default function ViewModal({userDetails,handleCloseView,showView}) {
+
+
+
+  return <>
+  
+ <div className="d-flex justify-content-center "> {/* Wrap modal within this div */}
```

Comment:
The style of the view modal still needs some adjustments and changes, as it doesn't look good currently.
![image](https://github.com/SalmaAboArab/Project-Management-System/assets/64327685/1af9b0b0-7bfc-46b0-9aad-bc84f2f1c984)